src/main/java/com/bittercode/service/impl/UserServiceImpl.java

```diff
@@ -84,7 +90,6 @@ public String register(UserRole role, User user) throws StoreException {
             int k = ps.executeUpdate();
             if (k == 1) {
                 responseMessage = ResponseCode.SUCCESS.name();
-                ;
```

Comment:
While I agree this `;` wasn't necessary, removing it probably belongs in a separate PR, since it has nothing to do with adding annotations. It's generally a good idea to keep each PR focused around only one task.